### PR TITLE
Lesson21

### PR DESCRIPTION
**Pull request by marcelomatosdev**

### CODE REVIEW

File: Lesson21/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+alert('Enter two numbers to see if the first is greater than the second');
+
+let firstNumber = prompt('Please enter the first number', 0);
+let secondNumber = prompt('Please enter the second number', 0);
```
Comment (anton-shevchook) [inline Lesson21/index.js]: It's better to convert prompt entered values into "number" type before converting.

**Review by anton-shevchook — CHANGES_REQUESTED** [review]
- [ ] [Convert to "number" type.](https://github.com/marcelomatosdev/comit-exercises/pull/5/files#r495494497)
Good that you used backtick template syntax,
you'll be using it often!
Please fix types, test it out and you can merge it.

**Review by marcelomatosdev — COMMENTED** [review]
Converted to Number

**Review by anton-shevchook — APPROVED** [review]
Good job!
You can merge it.

**Pull request merged**